### ISSUE TITLE
Discussion of how various chemicals in CHEBI are classified as lipids.

### ISSUE THREAD

**Issue by hdrabkin** [ebi-chebi/ChEBI#4176]
In a recent import of CHEBI into the Gene Ontology, a diff report showed several unexpected new inferences involving lipid metabolic processes. that we have not seen in the past:

Added
•	fumarate transmembrane transporter activity SubClassOf lipid transporter activity
Added
•	fumarate transport SubClassOf lipid transport
Added
•	negative regulation of tetrapyrrole biosynthetic process from glycine and succinyl-CoA SubClassOf negative regulation of lipid metabolic process
Added
•	succinate metabolic process SubClassOf lipid metabolic process
Added
•	tyrosine catabolic process to fumarate SubClassOf lipid biosynthetic process

1. We would NOT expect succinate or fumarate to be connected to lipid metabolic process.  However, they are both classified as 'fatty acids', and hence a lipid.  The  'functional chain' of fatty acid in these two cases seems is very short. 

Since the definition 'Lipids'  Chebi CHEBI:18059 Lipid is a loosely defined term for substances of biological origin that are soluble in nonpolar solvent, we looked at numerous resources having information of the solubility of these acids. 
Many web sites declare that succinic acid is water-soluble  (58g/l; wiki, although  83 g/1000 ml (25 °C, reported http://www.sciencemadness.org/smwiki/index.php/Succinic_acid)  and is not soluble in non-polar solvents like hexane. ; fumaric acid, however, is certainly less water-soluble (about 4.9g/l, wiki), and Chemical Book states 6.3 g/1000 mL (25 ºC), . It is soluble in ethanol, slightly soluble in water and ether, but insoluble in chloroform.
We are wonder if the classification as a lipid is ignoring the water solubility of these compounds. Does solubility in non-polar solvents preclude water solubility?
Lipidmaps also classifies succinic acid as a fatty acid, but has recently removed fumaric acid (reason not known)

2. We would also not expect an amino acid, tyrosine, to a lipid. We assume that we pick this up because tyrosine is a 'structural derivative 'of propionic acid which in turn is classified as a fatty acid, and thus a lipid. Would like clarification on what is a structural derivative; we are thinking that we should NOT reason over this relationship. 

We think that there may need to be some re-examination of some of these relationships.

**Comment by aimol** [comment]
Hi Harold,
Indeed the definition of a lipid is pretty vague and in this case the solubility was ignored and the compounds were classified following the LIPID MAPS classification. I have now deleted the relationship to a lipid for fumaric acid.
Regarding tyrosine, I agree with you,I would not expect that an amino acid is classified as a lipid, maybe need to ignore "structural derivative" relationships.

**Comment by amalik01** [comment]
Hi Harold,

As a chemist i personally wouldn't classify succinic acid as a lipid. This is something that the lipid community needs to decide and agree on. A few months ago, i contacted Lipid Maps and received the following response: 

'' I guess it both is and isn't a lipid depending on your point of view. As there's no agreed definition of a lipid, its impossible to tell. Succinic acid is  not produced biosynthetically in the same way as longer chain fatty acids, so I can see that it might not count as a lipid biosynthetically, but there's no cut off for when a dicarboxylic acid becomes long enough to be called 'fatty'

**Comment by hdrabkin** [comment]
We also see that even shorter (malonic acid) appear to be classified as a lipid in CHEBI (3 carbon)
Oxalic acid does not seem to be.;

**Comment by amalik01** [comment]
These relationships are not being added by the ChEBI team. @aimol (Rhea) is making these changes in ChEBI based on the Lipid Maps classification. We have had discussions about this in the past, i think this topic requires further discussion with the Lipids community since they need to decide what constitutes a lipid and what doesn't.

**Comment by aimol** [comment]
Hi Harold,
Oxalic, like Acetic has only 2 carbons and these are not considered lipids.

**Comment by aimol** [comment]
Correct, I added these relations based on LIPID MAPS classification. I have now removed the relationships for succinic and malonic. As discussed, it is hard to define what constitutes a lipid, if we follow the definition of lipid used in CHEBI that they must be soluble in nonpolar solvents, then we disregard LIPID MAPS classification and remove those relationships.
Cheers,
Lucila

**Comment by hdrabkin** [comment]
This is great. Can you tell me when these changes will be reflected in the public release CHEBI?

**Comment by amalik01** [comment]
ChEBI's next release will be on the 1st of Jan 2022. You can redo your import after this date.

**Comment by hdrabkin** [comment]
After updating the CHEBI import with the new release, 
https://github.com/geneontology/go-ontology/pull/22657

we still get these inference s
succinyl-CoA binding http://purl.obolibrary.org/obo/GO_0120226
Added
    succinyl-CoA binding SubClassOf lipid binding

succinyl-CoA biosynthetic process http://purl.obolibrary.org/obo/GO_1901290
Added
    succinyl-CoA biosynthetic process SubClassOf lipid biosynthetic process

succinyl-CoA catabolic process http://purl.obolibrary.org/obo/GO_1901289
added
    succinyl-CoA catabolic process SubClassOf lipid catabolic process

succinyl-CoA metabolic process http://purl.obolibrary.org/obo/GO_0006104
Added
    succinyl-CoA metabolic process SubClassOf lipid metabolic process

We believe this is because in CHEBI  succinyl-CoA (5-) is a child of both omega carboxyacyl-CoA(5-) and 
omega carboxy (fatty acyl) CoA (5-). The latter one is a child of 'lipid' whereas omega caboxyacyl-CoA(5-) is not. Can this be looked at?

**Comment by amalik01** [comment]
I have now removed the  ω-carboxy-(fatty acyl)-CoA(5−) relationship from this entry. If you think there are other issues with some of the other entries, then please do let me know. Thanks
